src/Tribe/Views/V2/Hooks.php

```diff
@@ -68,6 +69,35 @@ protected function add_actions() {
 		add_action( 'get_header', [ $this, 'print_single_json_ld' ] );
 		add_action( 'tribe_template_after_include:events/v2/components/after', [ $this, 'action_add_promo_banner' ], 10, 3 );
 		add_action( 'tribe_events_parse_query', [ $this, 'parse_query' ] );
+		add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_customizer_in_block_editor' ] );
+	}
+
+	/**
+	 * Enqueue Customizer styles for the single event block editor screen.
+	 *
+	 * @since TBD
+	 */
+	public function enqueue_customizer_in_block_editor() {
+		// Make sure we're on the block edit screen
+		if ( ! is_admin() || ! get_current_screen()->is_block_editor ) {
+			return;
+		}
```

Comment:
Let's ensure that the assets only load on our post type pages.

```suggestion
		}
		
		if ( ! tribe( 'admin.helpers' )->is_post_type_screen() ) {
			return;
		}
```